supabase/database.types.ts

```diff
@@ -0,0 +1,280 @@
+export type Json =
```

Comment:
These types are auto-generated via `supabase` CLI, based on table/schema & function definitions set up in Supabase (cloud project)